ext/pcntl/pcntl.stub.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+function pcntl_fork(): int {}
+
+function pcntl_waitpid(int $pid, &$status, int $options = 0, &$rusage = []): int {}
+
+function pcntl_wait(&$status, int $options = 0, &$rusage = []): int {}
+
+/**
+ * @param callable|int $handler
+ */
+function pcntl_signal(int $signo, $handler, bool $restart_syscalls = true): bool {}
+
+/** @return mixed */
+function pcntl_signal_get_handler(int $signo) {}
+
+function pcntl_signal_dispatch(): bool {}
+
+/**
+ * @param ?array $oldset
+ */
+function pcntl_sigprocmask(int $how, array $set, &$oldset = null): bool {}
+
+#ifdef HAVE_STRUCT_SIGINFO_T
+#if HAVE_SIGWAITINFO && HAVE_SIGTIMEDWAIT
+/** @return int|false */
+function pcntl_sigwaitinfo(array $set, &$info = []) {}
+
+/** @return int|false */
+function pcntl_sigtimedwait(array $set, &$info = [], int $seconds = 0, int $nanoseconds = 0) {}
+#endif
+#endif
+
+function pcntl_wifexited(int $status): bool {}
+
+function pcntl_wifstopped(int $status): bool {}
+
+#ifdef HAVE_WCONTINUED
```

Comment:
Side note: This is pretty weird. All the other ones are defined unconditionally and just return false.

Reply:
Presumably just a style thing as this method was submitted separately:
https://github.com/php/php-src/pull/1114
Shall I open a PR to make it consistent and just return `false`?

Reply:
Presumably just a style thing as this method was submitted separately:
https://github.com/php/php-src/pull/1114
Shall I open a PR to make it consistent and just return `false`?

Reply:
Yeah, I think that would be good.